src/Symfony/Component/Form/Test/DeprecationErrorHandler.php

```diff
@@ -24,7 +24,11 @@ public static function handle($errorNumber, $message, $file, $line, $context)
             return true;
         }
 
-        return \PHPUnit_Util_ErrorHandler::handleError($errorNumber, $message, $file, $line);
+        if (class_exists('\PHPUnit_Util_ErrorHandler')) {
+            return \PHPUnit_Util_ErrorHandler::handleError($errorNumber, $message, $file, $line);
+        }
+
+        return \PHPUnit\Util\ErrorHandler::handleError($errorNumber, $message, $file, $line);
```

Comment:
I personal have a preference to keep the FQCN inline with no use - makes things more readable.